test/pom.xml

```diff
@@ -191,6 +207,15 @@ THE SOFTWARE.
         <artifactId>maven-hpi-plugin</artifactId>
         <!-- Version specified in grandparent POM -->
         <extensions>true</extensions>
+        <executions>
+          <execution>
+            <id>test-runtime</id>
+            <goals>
+              <goal>test-runtime</goal>
+            </goals>
+            <phase>test</phase>
+          </execution>
+        </executions>
```

Comment:
Sets the `jenkins.addOpens` and `jenkins.insaneHook` system property. The latter is needed for the newer version of INSANE.